addons/udes_stock/tools/find_multi_location_packages_and_pallets.sql

```diff
@@ -0,0 +1,44 @@
+\timing
+
+-- Find packages in multiple locations
+SELECT
+    p.package_id,
+    COUNT(location_id)
+FROM (
+    SELECT
+        package_id,
+        location_id
+    FROM
```

Comment:
We can add a WHERE here to filter out quants without package_id, so we won't need it in the having.

Reply:
Changed